src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -73,7 +73,7 @@
                                           'relevant for listed add-ons.') }}">?</span>
   </li>
 {% endif %}
-{% if addon.type == amo.ADDON_EXTENSION and amo.FIREFOX in addon.compatible_apps %}
+{% if addon.is_listed and addon.type == amo.ADDON_EXTENSION and amo.FIREFOX in addon.compatible_apps %}
```

Comment:
Like above, it's a hack I added because `compatible_apps` depends on `current_version`. And yeah, like above, we need to decide something, but I figured it'd make more sense to only care about listed versions to avoid showing info about a potential unlisted version, making it confusing for the developer if we show a compatibility warning for their add-on in the public detail page.